### PR TITLE
staging build: fix up build tagging

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: dev/staging/push.sh [hunk 1]
```diff
@@ -20,7 +20,11 @@ set -x;
...
 cd "${REPO_ROOT}"
 
+if [[ -z "${VERSION}" ]]; then
+  VERSION=$(git describe --always)
+fi
+
 # Build addon bootstrap image
 pushd bootstrap
-make docker-image
+IMG=gcr.io/k8s-staging-cluster-api/cluster-addons-bootstrap:${VERSION} make docker-image
```
Comment (stealthybox) [inline dev/staging/push.sh:29]: Let's amend the Makefile to also be able to push

likely good to have a separate target for it that we can call here after `make docker-image`

**Comment by justinsb** [comment]
Now with push!

**Comment by stealthybox** [comment]
nice
/lgtm
/approve

**Pull request merged**